GTG/gtk/browser/taskbrowser.ui

```diff
@@ -277,38 +277,61 @@
-                  </object>
-                  <packing>
-                    <property name="expand">False</property>
-                    <property name="fill">False</property>
-                    <property name="padding">2</property>
-                    <property name="position">0</property>
-                  </packing>
+                        <property name="search-mode-enabled">False</property>
+                        <property name="show_close_button">False</property>
+                        <property name="valign">center</property>
+                        <child>
+                            <object class="GtkBox" id="view_box">
+                                <property name="visible">True</property>
+                                <property name="can_focus">False</property>
+                                <property name="has_focus">False</property>
+                                <property name="is_focus">False</property>
+                                <property name="spacing">6</property>
+                                <child>
+                                    <object class="GtkSearchEntry" id="search_entry">
+                                        <property name="visible">True</property>
+                                        <property name="can_focus">True</property>
+                                        <property name="has_focus">True</property>
+                                        <property name="is_focus">True</property>
+                                        <property name="can_default">True</property>
+                                        <property name="max-width-chars">40</property>
+                                        <property name="primary_icon_activatable">False</property>
+                                        <property name="placeholder-text">Search here</property>
+                                        <signal handler="on_search" name="key-release-event" swapped="no"/>
+                                    </object>
+                                    <packing>
+                                        <property name="expand">True</property>
+                                        <property name="fill">True</property>
+                                    </packing>
+                                </child>
+                                <child>
+                                    <object class="GtkButton" id="save_search">
+                                        <property name="visible">True</property>
+                                        <property name="label">Save Search</property>
+                                        <signal handler="on_save_search" name="clicked" swapped="no"/>
+                                    </object>
+                                </child>
+                                                            </object>
+                            <packing>
+                                <property name="primary_icon_activatable">False</property>
```

Comment:
re-set it to True, however I'm wondering if we need this property at all.

UPDATE: I removed the property entirely, as it had no effect on the app and moreover it was placed in a separate <packing></packing> block of code which was weird, too. Probably it was just some copy-paste error.